### PR TITLE
Script to help update chains

### PR DESCRIPTION
**Pull request by JasoonS**
image

This makes it much easier to add/remove chains. This is an intermediate solution until we implement dotfiles or similar.

### CODE REVIEW

**Review by DZakh — APPROVED** [review]
Looks good

**Comment by JonoPrest** [comment]
@DZakh I've moved it to the cli module. I think it will be helpful to add scripts/tools to the hidden "script" subcommand. Then it becomes easy to extend

File: codegenerator/cli/Makefile [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
-	cargo run -- print-config-json-schema fuel > ./npm/envio/fuel.schema.json
+	cargo run -- script print-config-json-schema fuel > ./npm/envio/fuel.schema.json
 
 # Target to update help
 update-help:
 	@echo "Updating help doc..."
-	cargo run -- print-cli-help-md > ./CommandLineHelp.md
+	cargo run -- script print-cli-help-md > ./CommandLineHelp.md
+
+# Target to update help
```
Comment (JonoPrest) [inline codegenerator/cli/Makefile:13]:
```suggestion
# Target to help find missing networks
```

**Review by DZakh — APPROVED** [review]
Good, I only don't like that it's a part of the published binary, but we can solve this later.

**Pull request merged**